tools/sdk/include/user_interface.h

```diff
@@ -27,9 +27,19 @@
 
 #include "os_type.h"
 #ifdef LWIP_OPEN_SRC
+
+#include "lwip/init.h"
+#if LWIP_VERSION_MAJOR == 1
+#define ipv4_addr ip_addr
+#endif
 #include "lwip/ip_addr.h"
+#if LWIP_VERSION_MAJOR != 1
+typedef struct ip4_addr ipv4_addr_t;
+#endif
+
 #else
 #include "ip_addr.h"
+#error
```

Comment:
I assume this means that we can no longer use non-open src. In that case the include just above the error doesn't make sense.